docs/UserGuide.md

```diff
@@ -280,6 +282,7 @@ Column Name | Explanation
 [Optional] Repository's Location | Same as `repo-config.csv`. Default: all the repos in `repo-config.csv`
 [Optional] Branch | The branch to analyze for this author e.g., `master`. Default: the default branch of the repo
 Author's GitHub ID | GitHub username of the target author e.g., `JohnDoe`
+[Optional] Author's Emails<sup>*</sup> | Associated Github emails of the author. This can be found in your [GitHub settings](https://github.com/settings/emails).
```

Comment:
Add ```Default``` for email here?

Reply:
There isn't any _default_ emails actually. The additional email I added is based off the author's github id and its consider more of an additional rather than default. It's an implementation that need not to be known by the users.